autodeebee.js

```diff
@@ -0,0 +1,86 @@
+const Hyperbee = require('hyperbee')
+const b4a = require('b4a')
+
+module.exports = class Autodeebee {
```

Comment:
Do you think you could publish this as a separate module which we could link to in the README for folks to test and include in the `devDependencies` for the tests?